### PR TITLE
WEB-644: fix css link, refactor demos, update script names

### PR DESCRIPTION
**Pull request by e-wipond**

### CODE REVIEW

File: checkoutfields/assets/js/form/view.js [hunk 1]
```diff
@@ -26,8 +26,8 @@
                 this.submitBtn = this.form.querySelector('button[type=submit]');
             }
 
-            var base = document.createElement('base');
-            this.host = base.baseURI;
+            var urlArray = this.currentScript.src.split('/');
+            this.host = urlArray[0] + '//' + urlArray[2] + '/' + urlArray[3] + '/' + urlArray[4] + '/';
```
Comment (dlight-bean) [inline checkoutfields/assets/js/form/view.js]: Add a comment to the code as to why your are splitting it this way

File: demos/dev/index.html [hunk 2]
```diff
@@ -47,10 +29,10 @@
                 </div>
 
                 <!--
-                Path set relative to host for demo. In production use absolute path:
+                In production use absolute path:
                 https://libs.na.bambora.com/checkoutfields/1.1.1/checkoutfields.js
                 -->
-                <script src='checkoutfields.js' id="checkoutfields-script"></script>
+                <script src='https://payform-dev.beanstream.com/checkoutfields/0.0.0/checkoutfields.js' id="checkoutfields-script"></script>
```
Comment (dlight-bean) [inline demos/dev/index.html]: url?

File: demos/master/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                    <label>CVV</label>
+                    <div data-bambora-target='ccCvv_input'></div>
+                    <div data-bambora-target='ccCvv_error' class='help-block'></div>
+                </div>
+
+                <!--
+                In production use absolute path:
+                https://libs.na.bambora.com/checkoutfields/1.1.1/checkoutfields.js
+                -->
+                <script src='https://payform-master.beanstream.com/checkoutfields/0.0.0/checkoutfields.js' id="checkoutfields-script"></script>
```
Comment (dlight-bean) [inline demos/master/index.html]: url?

File: demos/test/index.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                    <label>CVV</label>
+                    <div data-bambora-target='ccCvv_input'></div>
+                    <div data-bambora-target='ccCvv_error' class='help-block'></div>
+                </div>
+
+                <!--
+                In production use absolute path:
+                https://libs.na.bambora.com/checkoutfields/1.1.1/checkoutfields.js
+                -->
+                <script src='https://payform-test.beanstream.com/checkoutfields/0.0.0/checkoutfields.js' id="checkoutfields-script"></script>
```
Comment (dlight-bean) [inline demos/test/index.html]: url?

File: demos/dev/index.html (hunk 2 above)
Comment (e-wipond) [inline demos/dev/index.html]: URLs will stay the same as they are internal only and this version of the product will be depricated in the near future.

File: demos/master/index.html (hunk 3 above)
Comment (e-wipond) [inline demos/master/index.html]: See above.

File: demos/test/index.html (hunk 4 above)
Comment (e-wipond) [inline demos/test/index.html]: See above.

**Pull request merged**